tools/bioimaging/bioimage_inference.xml

```diff
@@ -0,0 +1,79 @@
+        <param name="input_image_file" type="data" format="tiff,png" label="Input image" help="Please provide an input image for the analysis."/>
+        <param name="input_image_input_size" type="text" label="Shape of the input image" help="Provide shape of input image. See chosen model's RDF file to find correct input shape. For example: for the BioImage.IO model MitochondriaEMSegmentationBoundaryModel, the input shape is 256 * 256 * 32 * 1. Enter the shape as 256,256,32,1."/>
+    </inputs>
+    <outputs>
+        <data format="tif" name="output_predicted_image" from_work_dir="output_predicted_image.tif" label="Predicted image"></data>
+        <data format="npy" name="output_predicted_image_matrix" from_work_dir="output_predicted_image_matrix.npy" label="Predicted image tensor"></data>
+    </outputs>
+    <tests>
+        <test>
+            <param name="input_imaging_model" value="input_imaging_model.zip" location="https://zenodo.org/api/records/6647674/files/weights-torchscript.pt/content"/>
+            <param name="input_image_file" value="input_image_file.tif" location="https://zenodo.org/api/records/6647674/files/sample_input_0.tif/content"/>
+            <param name="input_image_input_size" value="256,256,1,1"/>
+            <output name="output_predicted_image" file="output_nucleisegboundarymodel.tif" compare="sim_size" delta="100" />
+            <output name="output_predicted_image_matrix" file="output_nucleisegboundarymodel_matrix.npy" compare="sim_size" delta="100" />
+        </test>
+        <test>
+            <param name="input_imaging_model" value="input_imaging_model.zip" location="https://zenodo.org/api/records/6647674/files/weights-torchscript.pt/content"/>
+            <param name="input_image_file" value="input_nucleisegboundarymodel.png"/>
+            <param name="input_image_input_size" value="256,256,1,1"/>
+            <output name="output_predicted_image" file="output_nucleisegboundarymodel.tif" compare="sim_size" delta="100" />
+            <output name="output_predicted_image_matrix" file="output_nucleisegboundarymodel_matrix.npy" compare="sim_size" delta="100" />
+        </test>
+    </tests>
+    <help>
+        <![CDATA[
+**What it does**
+
+The tool takes a BioImage.IO model and an image (as TIF or PNG) to be analysed. The model makes a prediction, and the predicted image becomes available as a TIF file in the Galaxy history.
+
+**Input files**
+    - BioImage.IO model: Add one of the model from Galaxy file uploader by choosing a "remote" file at "ML Models/bioimaging-models"
+    - Image to be analysed: Provide an image as TIF/PNG file
+    - Provide the necessary input shape for the model. This information can be found in the RDF file of each model (RDF file > config > test_information > inputs > size)
+
+**Output files**
+    - Predicted image: Predicted image using the BioImage.IO model
+    - Predicted image matrix: Predicted image matrix in original dimensions
+        ]]>
```

Comment:
This indentation is still out of place. I have fixed BE→AE, and I have also added a sentence to briefly motivate what "prediction" is about (probably unknown by the target audience).
```suggestion
            **What it does**

            The tool takes a BioImage.IO model and an image (as TIF or PNG) to be analyzed. The analysis is performed by the model. The model is used to obtain a prediction of the result of the analysis, and the predicted image becomes available as a TIF file in the Galaxy history.

            **Input files**
            - BioImage.IO model: Add one of the model from Galaxy file uploader by choosing a "remote" file at "ML Models/bioimaging-models"
            - Image to be analyzed: Provide an image as TIF/PNG file
            - Provide the necessary input shape for the model. This information can be found in the RDF file of each model (RDF file > config > test_information > inputs > size)

            **Output files**
            - Predicted image: Predicted image using the BioImage.IO model
            - Predicted image matrix: Predicted image matrix in original dimensions
        ]]>
```